quality_of_service_demo/rclpy/quality_of_service_demo_py/message_lost_listener.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2020 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rclpy
+from rclpy.executors import SingleThreadedExecutor
+from rclpy.node import Node
+from rclpy.qos_event import SubscriptionEventCallbacks
+from rclpy.time import Time
+
+from sensor_msgs.msg import Image
+
+
+class MessageLostListener(Node):
+    """Listener node to demostrate how to get a notification on lost messages."""
```

Comment:
```suggestion
    """Listener node to demonstrate how to get a notification on lost messages."""
```

Reply:
Fixed in https://github.com/ros2/demos/pull/457/commits/c467247ee61b4ada2715d6a4a0b748ef5aeff6a4